pkg/scorecard.go

```diff
@@ -101,7 +102,47 @@ func RunScorecards(ctx context.Context,
 		Date: time.Now(),
 	}
 	resultsCh := make(chan checker.CheckResult)
-	go runEnabledChecks(ctx, repo, checksToRun, repoClient, ossFuzzRepoClient, ciiClient, resultsCh)
+	go runEnabledChecks(ctx, repo, nil, checksToRun, repoClient, ossFuzzRepoClient, ciiClient, resultsCh)
+	for result := range resultsCh {
+		ret.Checks = append(ret.Checks, result)
+	}
+	return ret, nil
+}
+
+// RunScorecardsRaw runs enabled Scorecard checks on a Repo
+// and returns raw results.
+func RunScorecardsRaw(ctx context.Context,
```

Comment:
Do we need a separate fn here? If `raw` bool is true, we simply need to pass a non-nil `RawResults` correct?

Reply:
you're right. I was hesitant to change the original function while we transition, to keep things as separate as possible. Shall I use a single function?

Reply:
Yeah I think single fn will be better.

Reply:
done.